Resources/Textures/Objects/Weapons/Guns/Launchers/china_lake.rsi/meta.json

```diff
@@ -10,15 +10,6 @@
     {
       "name": "icon"
     },
-    {
-      "name": "base"
```

Comment:
The other ones are base, but this one has icon left? Is this on purpose?

Reply:
Yes, I generally did that for stuff with no mag states.